CorsixTH/Lua/dialogs/resizables/menu_list_dialogs/custom_campaign.lua

```diff
@@ -33,42 +33,63 @@ local col_scrollbar = {
 local details_width = 280
 
 --! Collect the campaign levels at the provided path
---!param path (str) File system path to search.
---!return (array) The found levels, with some basic information about each level.
-local function createCampaignList(path)
-  local campaigns = {}
-
-  for file in lfs.dir(path) do
-    if file:match("%.campaign$") then
-      local campaign_info, err = TheApp:readCampaignFile(file)
-      if not campaign_info then
-        print(err)
+--!param paths_table (array) File system paths to search.
+--!return (array) The found levels, with some basic information about each level, in alphabetical order.
+local function createCampaignList(paths_table)
+  local campaigns, unique_names, duplicates = {}, {}
```

Comment:
Left side has 3 variables, right side has 2 values.
As it is now, it can be easily seen as a missing 3rd value.

If it is not, split the variable declarations so it is clear that it's not a mistake.